com.kavilgroup.gelectrical.json

```diff
@@ -0,0 +1,438 @@
+{
+    "app-id" : "com.kavilgroup.gelectrical",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "gelectrical_launcher.py",
+    "finish-args": [
+           "--socket=wayland",
+           "--socket=fallback-x11",
+           "--share=ipc",
+           "--filesystem=host"
```

Comment:
that's a very broad filesystem permission

Reply:
Actually I needed something like this

https://github.com/flatpak/xdg-desktop-portal/issues/463

Currently this permission seems to be the only option.